tests/unit/test-api.c

```diff
@@ -700,9 +701,25 @@ main (void)
   // cleanup.
   jerry_release_value (global_obj_val);
 
-  // TEST: run gc.
+  // Test: run gc.
   jerry_gc ();
 
+  // Test: number
+  val_t = jerry_create_number (6.25);
+  number_val = jerry_get_number_value (val_t);
+  TEST_ASSERT (number_val * 3 == 18.75);
+  jerry_release_value (val_t);
+
+  val_t = jerry_create_number_nan ();
+  number_val = jerry_get_number_value (val_t);
+  TEST_ASSERT (number_val != number_val);
+  jerry_release_value (val_t);
+
+  val_t = jerry_create_number_infinity (true);
+  number_val = jerry_get_number_value (val_t);
+  TEST_ASSERT (number_val * 3 == number_val);
```

Comment:
In the unlikely event of `number_val == 0.0`, the assert would still pass, which is not what we want. Thus, I'd rewrite this checks as
```c
TEST_ASSERT (number_val * 3 == number_val && number_val != 0.0);
```